sync/clients/databricks.py

```diff
@@ -146,13 +147,13 @@ def _send(self, request: httpx.Request) -> dict:
         if response.headers.get("Content-Type", "").startswith("application/json"):
             response_json = response.json()
             if "error_code" in response_json:
-                # Though not in the documentation, the cluster API can return and "error_code" too
-                # return {"error": {"code": "Databricks API Error", "message": f"{response_json['error_code']}: {response_json.get('message')}"}}
                 return response_json
 
-        # return {"error": {"code": "Databricks API Error", "message": "Transaction failure"}}
         logger.error(f"Unknown error - {response.status_code}: {response.text}")
-        return {"error_code": "UNKNOWN_ERROR", "message": "Transaction failure"}
+        return {
+            "error_code": str(response.status_code),
+            "message": httpx.codes.get_reason_phrase(response.status_code),
+        }
```

Comment:
Another improvement copied over from the backend - might as well send back the error code